cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/AbstractTicket.java

```diff
@@ -151,12 +153,29 @@ protected boolean isExpiredInternal() {
     }
 
     @Override
-    public final int hashCode() {
-        return new HashCodeBuilder(13, 133).append(this.getId()).toHashCode();
+    public final String toString() {
+        return this.getId();
     }
 
     @Override
-    public final String toString() {
-        return this.getId();
+    public boolean equals(final Object o) {
```

Comment:
It made sense to move it up since it was an opportunity to unify the methods. I'll reverse this and stick to making them all function similarly.